source/extensions/common/wasm/context.cc

```diff
@@ -146,7 +146,10 @@ Context::Context(Wasm* wasm, const PluginSharedPtr& plugin) : ContextBase(wasm,
   root_local_info_ = &std::static_pointer_cast<Plugin>(plugin)->local_info_;
 }
 Context::Context(Wasm* wasm, uint32_t root_context_id, const PluginSharedPtr& plugin)
-    : ContextBase(wasm, root_context_id, plugin) {}
+    : ContextBase(wasm, root_context_id, plugin) {
+  ENVOY_LOG(trace, "wasm log{}: Context::Context(..., root_context_id={}, ...)", log_prefix(),
```

Comment:
`wasm log` prefix is used for logs generated from within WasmVM, so adding it for things logged from outside the WasmVM is a bit confusing.